addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -509,8 +514,18 @@ $truncate-limits: 2, 3, 10;
             }
         }
     }
+
+    .o_wslides_undone_button.disabled,
+    .o_wslides_done_button.disabled {
+        // Show title attribute on disabled link
+        color: white!important;
+        pointer-events: auto;
+    }
 }
 
+.o_wslides_lesson_header_container {
+    height: 80px;
+}
```

Comment:
We could perhaps add a bottom padding when the slide is completed in the template class instead ?

Reply:
this is to have a fixed height whatever the progress bar is visible or not, so indeed the padding might fix it, but it will not be as robust as this (if the progress bar height change, we need to think to update the padding which is a bit bricolage)